contrib/dracut/02zfsexpandknowledge/module-setup.sh.in

```diff
@@ -77,10 +69,9 @@ check() {
     local _depdev
     local _depdevname
     local _depdevtype
-    local _depmajmin
-    local _dev
 
-if [[ $hostonly ]]; then
+# shellcheck disable=SC2154
+if [ -e "$hostonly" ]; then
```

Comment:
Quoth dracut.modules(7):
> If the $hostonly variable is set, then the module check() function should be in "hostonly" mode, which means, that

Reply:
Apologies if I missed it, but my reading the docs leads me to believe that the variable should be either set or unset. I didn't get the impression that it should be set to a filename which may or may not exist. Why do we want to test with -e here, rather than -n?

Reply:
oh, you're right of course, i just somehow overfit the -e to -n when reading it every single time. updated, thanks